manifests/config.pp

```diff
@@ -38,6 +38,30 @@
     mode    => '0640',
   }
 
+  service { 'supervisord':
+    ensure => running,
+    enable => true,
```

Comment:
should there be any ordering on the service? Should it start after the package is installed? Or after the config file?

Reply:
fixed